### ISSUE TITLE
Slaves replikation bricht ab

### ISSUE THREAD

**Issue by ale2317** [cybertec-postgresql/walbouncer#5]
Nachdem wir den WAL Bouncer nun ca. 2 Wochen laufen haben, sind alle angeschlossenen Slaves nicht mehr funktionsfähig. 

Auslöser für das Problem scheint das Umbenennen und das neu Anlegen einer Datenbank gewesen zu sein. 

Der wal receiver process läuft auf dem Slave zwar weiter und repliziert, es kann sich allerdings kein User anmelden:

/usr/local/pgsql/bin/psql -h127.0.0.1 -p5432 -Upostgres -W
Password for user postgres: *****************
psql: FATAL:  could not read block 0 in file "global/166491417": read only 0 of 8192 bytes
FATAL:  could not read block 0 in file "global/166491417": read only 0 of 8192 bytes

Ich habe noch einen Slave im Fehlerzustand laufen. Wenn sich das mal jemand anschauen will bin ich per Skype zu erreichen.

**Comment by kmoppel** [comment]
Summary of the problem in english:

After renaming a database and recreating it, one cannot connect to the walbounced replica anymore:
FATAL: could not read block 0 in file "global/166491417": read only 0 of 8192 bytes

**Comment by kmoppel** [comment]
Could not repeat the error condition though. @ale2317 any extra infos would be useful here - was the renamed/recreated database part inclusion/exclusion in the config? What version of Postgres and Walbouncer was used?

**Comment by kmoppel** [comment]
ok, seems there's indeed problem with renaming databases - "alter database rename"  xlog entries are filtered out. will look for a fix:)

**Comment by kmoppel** [comment]
after a small investigation seems it's actually a "feature" as described in http://static.cybertec.at/wp-content/uploads/walbouncer.pdf. sadly on creating/renaming databases one needs to take WB offline. will add info about it to the README also.

**Comment by kmoppel** [comment]
I have now updated the documentation in https://github.com/cybertec-postgresql/walbouncer/commit/0e72af2e40cd3fad546473539ba43593ce37b4f2
As said in the README, renaming databases is sadly a bit clumsy currently, one needs to take a couple of steps. What could be improved with less work though is automatic reload of config on client connect, so I'll create a separate issue for that. Closing this ticket for now.
